src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -22,12 +21,12 @@ object TezosTypes {
     private val script = GenLens[Origination](_.script)
     private val parameters = GenLens[Transaction](_.parameters)
 
-    private val storage = GenLens[Contracts](_.storage)
-    private val code = GenLens[Contracts](_.code)
+    private val storage = GenLens[Scripted.Contracts](_.storage)
```

Comment:
Could we use this occasion to leave a doc comment for `Scripted`?

Reply:
When I started doing 1-on-1 conversion of tezos schema definitions to Scala types, I sticked as much as possible to the original schema structure, so that it should be much easier to figure out how the two maps to each other.
In the operations' schema this sub-object was defined as Scripted.Contracts.

The upside is that we should be able to better understand what might be broken if a new protocol changes the schema and the json decoder stops parsing correctly some piece of data.

I don't know if this is what you meant or if you were talking about something else here